### PR TITLE
add host, port, discovery and configuration options to Konnected component

### PR DESCRIPTION
**Pull request by heythisisnate**
**Description:**
Document the new config options in the latest Konnected component iteration.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant):** https://github.com/home-assistant/home-assistant/pull/17120

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follow the [standards][standards].

### CODE REVIEW

**Review by frenck — DISMISSED** [review]

**Comment by frenck** [comment]
Thanks, @heythisisnate!

✅ Approved. Can be merged as soon as the parent PR gets merged.

File: source/_components/konnected.markdown [hunk 1]
```diff
@@ -116,6 +116,23 @@ devices:
...
           required: false
+    host: 
+      type: string
+      required: false
+      description: Optionally specify the Konnected device's IP address or hostname to set up without discovery.
+    port:
+      type: integer
+      required: false
+      description: Optionally specify the port number for the Konnected API on the device. Note that the port is different on every device. See help.konnected.io to learn how to determine the port number.
+    discovery:
```
Comment (tjorim) [inline source/_components/konnected.markdown:127]: Add the default values to this section.
For discovery `default: true`

File: source/_components/konnected.markdown [hunk 2]
```diff
@@ -116,6 +116,23 @@ devices:
...
+      type: boolean
+      required: false
+      description: Enable or disable discovery for this device. When true, the device will respond to discovery requests on your network. When false, the device will not respond to discovery requests, so it's important that you set reserved IP for the device and configure the _host_ and _port_ here.
```
Comment (tjorim) [inline source/_components/konnected.markdown]: Use code quotes (that backwards single quote thing) when talking about boolean values in the description to increase readability:

- true  --> \`true\`
- false --> \`false\`

File: source/_components/konnected.markdown [hunk 3]
```diff
@@ -116,6 +116,23 @@ devices:
...
+    blink:
```
Comment (tjorim) [inline source/_components/konnected.markdown:132]: `default: true` as well

**Review by tjorim — CHANGES_REQUESTED** [review]
The parent PR has been merged!

Please reverse the order of `type`, `required` and `description` to be in line with the other documentation. Other than that, only some small changes needs to be done here.

**Review by frenck — APPROVED** [review]
Updated the PR. Thanks for the review @tjorim. Please note, there is no standard on the order of the configuration fields.

Can be merged as soon as the build passes.

**Pull request merged**